cmd/files.go

```diff
@@ -13,6 +14,41 @@ type Updater interface {
 	Update(y, yhat string)
 }
 
+type JsonMetrics struct {
+	ID        string
+	TP        int64
+	FP        int64
+	TN        int64
+	FN        int64
```

Comment:
I'm not sure how you feel about this, but you could embed the whole ConfusionMatrix struct inside the JsonMetrics struct and all those fields would get put inside here.

Reply:
Comment addressed in latest commit.